### ISSUE TITLE
Post code/city inputs should be on separate lines

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#20409]
It's hard to tell at 200% text if there are 2 inputs (which is which?) or just one called Postal City Code that has a weird wraparound.

<img width="318" height="551" alt="Image" src="https://github.com/user-attachments/assets/c6802e62-9f62-44dd-a59e-1e4cadf7b924" />

### How to reproduce

In Firefox go to Preferences > General > Zoom
- Ensure default Zoom is set to 100%
- check Zoom text only

Create shop with 1 product

Visit storefront homepage
- Set viewport to 320px wide
- Zoom text to 200%
- Add product to cart
- Proceed to checkout > Register page
